DemoProject/DemoProject/WebServices/UserListAlamoFireViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  UserListAlamoFireViewController.swift
+//  DemoProject
+//
+//  Created by Devarsh Bhalara on 15/06/23.
+//
+
+import UIKit
+
+class UserListAlamoFireViewController: UIViewController {
+    
+    //MARK: Outlets
+    @IBOutlet weak var tableView: UITableView!
+    
+    //MARK: Variable
+    let url = "https://reqres.in/api/users?page=2"
```

Comment:
Why URL is declared here and not commonly using API endpoints?

Reply:
I had declared here because at that i didn't implement that network layer